packages/jest-runner/worker.ts

```diff
@@ -82,24 +102,22 @@ export class JazzerWorker {
 		globalThis.beforeEach = circus.beforeEach;
 		// @ts-ignore
 		globalThis.afterEach = circus.afterEach;
-
-		// @ts-ignore
-		globalThis.it.fuzz = fuzz;
-		// @ts-ignore
-		globalThis.test.fuzz = fuzz;
 	}
 
 	async run(test: Test, config: Config.GlobalConfig) {
 		JazzerWorker.#currentTestPath = test.path;
 		await this.initialize(test);
 
 		const state = await this.loadTests(test);
+		const adjustedPattern = this.adjustTestPattern(
```

Comment:
Why are we now unconditionally removing $ from the test pattern? We have restricted it to describe block because `it.fuzz` is handled as a test block by intellij. With this commit if we have two tests `My test` and `My test 2`, and the user wants to run `My test`, removing the dollar sign will cause the runner to run both tests, which should not be the case

Reply:
Thx, didn't remember that situation. Reverted the change.